src/netgearpy/netgear.py

```diff
@@ -102,6 +103,30 @@ async def login(self, username: str, password: str) -> None:
             "DeviceConfig", "SOAPLogin", LOGIN_BODY.format(username, password)
         )
 
+    async def get_attached_devices(self) -> list[AttachedDevice]:
+        """Get the attached devices from the Netgear router."""
+        response = await self._post_xml(
+            "DeviceInfo",
+            "GetAttachedDevices",
+            GET_ATTACHED_DEVICES_BODY,
+        )
+        root = ET.fromstring(response)  # noqa: S314
+        for item in root.iter():
+            if item.tag != "NewAttachDevice":
+                continue
+            assert item.text is not None  # noqa: S101
+            response = item.text
+            break
+        devices = []
+        first = True
+        for entry in response.split("@"):
+            if first:
```

Comment:
This "first" looks rather ugly and fragile, why do we need to skip the first one?

Reply:
Aaah right, maybe use index [1::] instead of the bool to filter out the first?